### DISCUSSION TITLE
KneeBraces

### DISCUSSION THREAD

**Discussion by Terre-ronce** [JWock82/Pynite#209] [Q&A]
Hi,

I was wondering if there is a tick in PyNite to add kneebraces reinforcement in a frame (see image below). Since it is not yet possible (I think) to add variable section on a same membre i tried adding kneebrace by discreditizing the elemnt i want to reinforce and add different sections to each sub-element. However, since my frame is hyperstatic, this method doesn't account for the extra stiffness that the kneebrace is supposed to add, thus resulting in false results in the analysis for the ractions along the x_axis. 

If somebody has encounterred the same issue with PyNite or another software/library and has a way to make it work, or if somebody has any suggestion, it would be great for me. 

Thanks in advance.

![image](https://github.com/user-attachments/assets/79ea32fc-ceb3-43e9-9435-5af514f77233)

**Comment by JWock82** [comment]
`Pynite` doesn't handle non-prismtic members. The closest thing I can think of is to discretize the member as you've already done, but it sounds like that has it's issues too.
